src/models/query.js

```diff
@@ -0,0 +1,58 @@
+import QueryAggregate from '../../src/models/queryAggregate';
+
+class Query {
+    /**
+     * [constructor]
+     * @param  {String} name
+     * @return {Query Instance}
+     */
+    constructor(name = '') {
+      this.aggregates = [];
+      this.filters = [];
+      this.name = name;
+    }
+
+    /**
+     * [addAggregate]
```

Comment:
remove automated generated description. You should Keep it only if you turn into something useful.
Like describing what the functions does but for a straightforward method like this, it's not needed